cernopendata_client/cli.py

```diff
@@ -73,37 +73,33 @@ def get_metadata(server, recid, doi, title, output_fields):
     \b
     Examples:
       $ cernopendata-client get-metadata --recid 1
-      $ cernopendata-client get-metadata --recid 1 | jq -S '.metadata.title'
+      $ cernopendata-client get-metadata --recid 1 system_details.global_tag
     """
     validate_server(server)
     if recid is not None:
         validate_recid(recid)
     record_json = get_record_as_json(server, recid, doi, title)
+    output_json = record_json["metadata"]
     if output_fields is not None:
-        output_fields = [f.strip() for f in output_fields.split(",")]
-    if output_fields and len(output_fields) > 0:
-        try:
-            output_json = {field: record_json[field] for field in output_fields}
-        except KeyError:
-            top_level_fields = ", ".join(field for field in record_json)
+        fields = output_fields.split(".")
```

Comment:
The docs promises the output can be a list of wanted things, so note that `output_fields` can be a list itself.  The PR does currently this:

```console
$ cernopendata-client get-metadata --recid 5500 --output-fields recid,title
Key 'recid,title' is not present in metadata
```

One would expect to have listed both fields in the output.